native/dlahqr.go

```diff
@@ -25,7 +24,7 @@ import (
 //  H[ilo,ilo-1] == 0,  if ilo > 0,
 // otherwise Dlahqr will panic.
 //
-// wr[ilo:ihi+1] and wi[ilo:ihi+1] will contain, on return if converged is true,
+// If unconverged is zero, wr[ilo:ihi+1] and wi[ilo:ihi+1] will contain on return
```

Comment:
Ah, reading the comment in email I missed your suggested edit and misunderstood the message. Will send as a separate PR.